api/client/lib.py

```diff
@@ -452,6 +452,25 @@ def get_data_series(access_token, api_host, **selection):
         raise Exception(resp.text)
 
 
+def get_source_ranking(access_token, api_host, series):
```

Comment:
@nemozen Based on our conversation on chat, do we want to add back the reference in Client to this method?
`  def get_source_ranking(self, series):
        return lib.get_source_ranking(self.access_token, self.api_host, series)
`